pkg/controller/user/create.go

```diff
@@ -96,17 +98,35 @@ func (c *Controller) HandleCreate() http.Handler {
 			return
 		}
 
+		if !alreadyExists {
+			pwd, err := otp.GenerateAlphanumericCode(24)
+			if err != nil {
+				flash.Alert("Error generating password for '%v'", form.Email)
+				c.renderNew(ctx, w, user, false)
+				return
+			}
+
+			fbUser := &auth.UserToCreate{}
+			fbUser.Email(user.Email).DisplayName(user.Name).Password(pwd)
+			_, err = c.client.CreateUser(ctx, fbUser)
+			if err != nil {
+				flash.Alert("Error creating user '%v'", form.Email)
```

Comment:
I think there's an edge case where where a user exists in Firebase but not in our database. In that case, alreadyExists would be false (thus executing this code), but this code would fail because the user exists in firebase.

This can occur when a user is deleted from their last realm. I believe we auto-delete the user record after that, but we don't delete the firebase user.